### PR TITLE
Changelogs for Digital Twin Manager UI and Microservice 1020.1.15

### PR DESCRIPTION
**Pull request by ghost**
This PR contains the changelogs for Digital Twin Manager 1020.1.15

### CODE REVIEW

File: content/change-logs/application-enablement/dtm-1020.1.15-fixed-ui-issues-for-sample-asset-models.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: "Fixed UI issues for sample asset models"
```
Comment (lra-sag) [inline content/change-logs/application-enablement/dtm-1020.1.15-fixed-ui-issues-for-sample-asset-models.md:3]:
```suggestion
title: "Improved naming and consistency in Asset model samples"
```

File: content/change-logs/application-enablement/dtm-1020.1.15-fixed-ui-issues-for-sample-asset-models.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+product_area: "Application enablement & solutions"
+change_type:
+    - value: "change-VSkj2iV9m"
+      label: "Fix"
```
Comment (lra-sag) [inline content/change-logs/application-enablement/dtm-1020.1.15-fixed-ui-issues-for-sample-asset-models.md:7]:
```suggestion
    - value: "change-2c7RdTdXo4"
      label: "Improvement"
```

File: content/change-logs/application-enablement/dtm-1020.1.15-fixed-ui-issues-for-sample-asset-models.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+    - value: "component-Tl88RYb4A"
+      label: "Digital Twin Manager"
+build_artifact:
+    - value: "tc-wYIY0MBDO"
+      label: "dtm"
+ticket: "CTM-1512"
+version: "1020.1.15"
+---
+Updated the naming convention by removing "model" from the names of newly added sample asset models for better readability and uniformity. Also applied CSS to ensure descriptions in the sample asset model card are truncated with an ellipsis for consistency.
```
Comment (lra-sag) [inline content/change-logs/application-enablement/dtm-1020.1.15-fixed-ui-issues-for-sample-asset-models.md:17]:
```suggestion
Removed "model" from the asset model sample names for simplicity. Improved the description and UI layout to maintain consistency.
```

File: content/change-logs/application-enablement/dtm-1020.1.15-rename-microservice-dtm-ms-to-dtm.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: "rename microservice "dtm-ms" to "dtm""
```
Comment (mvoigt-sag) [inline content/change-logs/application-enablement/dtm-1020.1.15-rename-microservice-dtm-ms-to-dtm.md:3]:
```suggestion
title: "Renamed microservice from "dtm-ms" to "dtm""
```

File: content/change-logs/application-enablement/dtm-1020.1.15-rename-microservice-dtm-ms-to-dtm.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+product_area: "Application enablement & solutions"
+change_type:
+    - value: "change-2c7RdTdXo4"
+      label: "Improvement"
```
Comment (mvoigt-sag) [inline content/change-logs/application-enablement/dtm-1020.1.15-rename-microservice-dtm-ms-to-dtm.md:7]:
```suggestion
      label: "API change"
```

**Review by mvoigt-sag — COMMENTED** [review]
I added two suggestions rgd the log for the name change of the microservice

File: content/change-logs/application-enablement/dtm-1020.1.15-removal-of-asset-models-and-asset-model-samples-tabs-from-the-new-asset-models-screen.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: "Removal of asset models and asset model samples tabs from the new asset models screen"
```
Comment (lra-sag) [inline content/change-logs/application-enablement/dtm-1020.1.15-removal-of-asset-models-and-asset-model-samples-tabs-from-the-new-asset-models-screen.md:3]:
```suggestion
title: "Simplified **New asset models** page"
```

File: content/change-logs/application-enablement/dtm-1020.1.15-removal-of-asset-models-and-asset-model-samples-tabs-from-the-new-asset-models-screen.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+product_area: "Application enablement & solutions"
+change_type:
+    - value: "change-VSkj2iV9m"
+      label: "Fix"
```
Comment (lra-sag) [inline content/change-logs/application-enablement/dtm-1020.1.15-removal-of-asset-models-and-asset-model-samples-tabs-from-the-new-asset-models-screen.md:7]:
```suggestion
    - value: "change-2c7RdTdXo4"
      label: "Improvement"
```

File: content/change-logs/application-enablement/dtm-1020.1.15-removal-of-asset-models-and-asset-model-samples-tabs-from-the-new-asset-models-screen.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+component:
+    - value: "component-Tl88RYb4A"
+      label: "Digital Twin Manager"
+build_artifact:
+    - value: "tc-wYIY0MBDO"
+      label: "dtm"
+ticket: "CTM-1554"
+version: "1020.1.15"
+---
+The "Asset models" and "Asset model samples" tabs, which were displayed at the top of the New Asset Models screen, have been removed.
```
Comment (lra-sag) [inline content/change-logs/application-enablement/dtm-1020.1.15-removal-of-asset-models-and-asset-model-samples-tabs-from-the-new-asset-models-screen.md:17]:
```suggestion
In the **New asset model page**, removed the **Asset models** and **Asset model samples** tabs for a more focused experience .
```

File: content/change-logs/application-enablement/dtm-1020.1.15-removal-of-asset-models-and-asset-model-samples-tabs-from-the-new-asset-models-screen.md [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+component:
+    - value: "component-Tl88RYb4A"
+      label: "Digital Twin Manager"
+build_artifact:
+    - value: "tc-wYIY0MBDO"
+      label: "dtm"
+ticket: "CTM-1554"
+version: "1020.1.15"
+---
+In the **New asset model page**, removed the **Asset models** and **Asset model samples** tabs for a more focused experience .
```
[the remark this answers is not in the document]
Comment (lara-c8y) [reply content/change-logs/application-enablement/dtm-1020.1.15-removal-of-asset-models-and-asset-model-samples-tabs-from-the-new-asset-models-screen.md:17]: The tabs were part of feature implementation and this info was not added separately in the changelogs.

File: content/change-logs/application-enablement/dtm-1020.1.15-rename-microservice-dtm-ms-to-dtm.md [hunk 10]
```diff
@@ -0,0 +1,17 @@
...
+component:
+    - value: "component-Tl88RYb4A"
+      label: "Digital Twin Manager"
+build_artifact:
+    - value: "tc-wYIY0MBDO"
+      label: "dtm"
+ticket: "CTM-1535"
+version: "1020.1.15"
+---
+The  DTM microservice is renamed from "dtm-ms" to "dtm". REST endpoint paths have been updated accordingly, replacing the "dtm-ms" segment with "dtm". This change is transparent to the user. However, user may need to manually unsubscribe and uninstall the older version.
```
[the remark this answers is not in the document]
Comment (lara-c8y) [reply content/change-logs/application-enablement/dtm-1020.1.15-rename-microservice-dtm-ms-to-dtm.md:17]: DTM application users will not notice this change.

File: content/change-logs/application-enablement/dtm-1020.1.15-rename-microservice-dtm-ms-to-dtm.md [hunk 11]
```diff
@@ -0,0 +1,17 @@
...
+component:
+    - value: "component-Tl88RYb4A"
+      label: "Digital Twin Manager"
+build_artifact:
+    - value: "tc-wYIY0MBDO"
+      label: "dtm"
+ticket: "CTM-1535"
+version: "1020.1.15"
+---
+The  DTM microservice is renamed from "dtm-ms" to "dtm". REST endpoint paths have been updated accordingly, replacing the "dtm-ms" segment with "dtm". This change is transparent to the user. However, users may need to manually unsubscribe and uninstall the older version.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/dtm-1020.1.15-rename-microservice-dtm-ms-to-dtm.md:17]:
```suggestion
The  DTM microservice has been renamed from "dtm-ms" to "dtm". REST endpoint paths have been updated accordingly, replacing the "dtm-ms" segment with "dtm". Users will not perceive any change. However, they may need to manually unsubscribe and uninstall the older version.
```

File: content/change-logs/application-enablement/dtm-1020.1.15-removal-of-asset-models-and-asset-model-samples-tabs-from-the-new-asset-models-screen.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (BeateRixen) [reply content/change-logs/application-enablement/dtm-1020.1.15-removal-of-asset-models-and-asset-model-samples-tabs-from-the-new-asset-models-screen.md]: I mean, if the tabs have been removed where is the content that has been provided on the tabs now available for users?

[the remark this answers is not in the document]
Comment (lara-c8y) [reply content/change-logs/application-enablement/dtm-1020.1.15-removal-of-asset-models-and-asset-model-samples-tabs-from-the-new-asset-models-screen.md]: The tabs are already part of Asset models page. It was also present in New asset model page which we have now removed.

[the remark this answers is not in the document]
Comment (BeateRixen) [reply content/change-logs/application-enablement/dtm-1020.1.15-removal-of-asset-models-and-asset-model-samples-tabs-from-the-new-asset-models-screen.md]: Shouldn´t this then be mentioned? If I as a user read, that tabs have been removed, I wonder, where I can find its content now.

**Review by BeateRixen — APPROVED** [review]

File: content/change-logs/application-enablement/dtm-1020.1.15-removal-of-asset-models-and-asset-model-samples-tabs-from-the-new-asset-models-screen.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (BeateRixen) [reply content/change-logs/application-enablement/dtm-1020.1.15-removal-of-asset-models-and-asset-model-samples-tabs-from-the-new-asset-models-screen.md]: I would perhaps add something like "have been removed for a more focused user experience as they are not required for creating a new asset model."

[the remark this answers is not in the document]
Comment (lra-sag) [reply content/change-logs/application-enablement/dtm-1020.1.15-removal-of-asset-models-and-asset-model-samples-tabs-from-the-new-asset-models-screen.md]: Incorporated the suggestion

**Comment by BeateRixen** [comment]
@lra-sag @pavanshivajirao Can this PR be closed?

**Pull request closed without merging**